### DISCUSSION TITLE
NHK G and NHK E has disappeared :(

### DISCUSSION THREAD

**Discussion by macaufighter** [iptv-org/community#34] [Q&A] [answered]
Hi, does anybody know why both NHK G and NHK E have disappeared ? Is it possible to get an older version of the m3u file ? Thanks!

**Comment by BellezaEmporium** [comment, marked as answer]
Server down.

**Reply by macaufighter** [reply]
That's too bad. The streams were excellent. Hopefully they will return.

**Reply by BellezaEmporium** [reply]
Someone has found the new link and how they operate. Will be changed when I'm available.

**Reply by meowiamgithub** [reply]
Have you gotten it??

**Reply by BellezaEmporium** [reply]
Well, it has been a while since I've looked over it, was updated but died months after

**Comment by Alstruit** [comment]
See iptv-org/iptv-org.github.io#202 and PR #9032
